fizzbuzz.js

```diff
@@ -1,9 +1,68 @@
 // This is our main function
 function fizzbuzz() {
-    console.log("Hello, World!");
+    // Iterate from 1 to 255
+    for (let i = 1; i < 256; i++) {
+        // Initialise array
+        let answer = [];
+        if (i % 11 == 0) {
+            // Case where number is a multiple of 11
+            answer.push("Bong");
+        } else {
+            // Append strings for numbers that are identified multiples
+            answer = add_phrase(i, answer, 3, "Fizz");
+            answer = add_phrase(i, answer, 5, "Buzz");
+            answer = add_phrase(i, answer, 7, "Bang");
+        }
 
-    // Put your code here...
+        // Check further multiples
+        answer = insert(i, answer, 13, "Fezz", "B");
+        answer = reverse_phrase(i, answer, 17);
 
+        // Print number or phrase
+        if (answer.length > 0) {
+            // Case where number is a multiple
+            console.log(answer.join(''));
+        } else {
+            // Case where number is not an identified multiple
+            console.log(i);
+        }
+    }
+}
+
+// Function to check for multiple
+// Appends string to array
+function add_phrase(i, answer, n, phrase) {
```

Comment:
I like how you've commonized and refactored the rules into functions. It's not totally clear from the function name what `i` and `n` are though - can you think of a way to make it clearer that this is used to check the predicate for the rule (as opposed to `answer`, `phrase`, which are passed through for adding the phrase?